Assets/HoloToolkit/Utilities/Editor/ConfigureMenu.cs

```diff
@@ -22,34 +23,186 @@ public static void ShowHelp()
             Application.OpenURL("https://developer.microsoft.com/en-us/windows/holographic/unity_development_overview#Configuring_a_Unity_project_for_HoloLens");
         }
 
-        /// <summary>
-        /// Applies recommended scene settings to the current scenes
-        /// </summary>
-        [MenuItem("HoloToolkit/Configure/Apply HoloLens Scene Settings", priority = 0)]
-        public static void ApplySceneSettings()
+        [MenuItem("HoloToolkit/Configure/Apply All HoloLens Settings", priority = 0)]
+        public static void ApplyAllSettings()
         {
+            var output = new StringBuilder();
+            ApplySceneSettingsInternal(output);
+
+
+            var reload = ApplyProjectSettingsInternal(output);
+
+            EditorUtility.DisplayDialog("Results", output.ToString(), "Ok");
+
+            if (reload)
+            {
+                PromptForReload();
+            }
+        }
+
+
+        private static void ApplySceneSettingsInternal(StringBuilder output)
+        {
+            //todo if we find more than one camera, hololens will always use MainCamera (should)
+            //additional cameras could be used from a RenderTexture, etc. Should we warn?
+
+            output.AppendLine("** Checking Camera Settings");
+            Camera[] cameras = new Camera[Camera.allCamerasCount];
+            Camera.GetAllCameras(cameras);
+
             if (Camera.main == null)
             {
-                Debug.LogWarning(@"Could not apply settings - no camera tagged with ""MainCamera""");
+                output.AppendLine(@"Could not apply camera settings - no camera tagged with ""MainCamera""");
                 return;
             }
 
-            Camera.main.transform.position = Vector3.zero;
-            Camera.main.clearFlags = CameraClearFlags.SolidColor;
-            Camera.main.backgroundColor = Color.black;
-            Camera.main.nearClipPlane = 0.85f;
+            SetCameraDefaults(Camera.main, true, output);
+
+            //All done with MainCamera
+            cameras = cameras.Where(o => o != Camera.main).ToArray();
+
+            //Check remaining cameras
+            if (cameras.Length > 0)
+            {
+                foreach (var camera in cameras)
+                {
+                    Debug.Log(camera);
+                    var cameraDetails = string.Format("Update this camera for HoloLens defaults? \r\nThis will set position to 0,0,0 (on MainCamera only) and background color to Clear Flags:Solid color(Black). \r\n \r\n---------------\r\nName: {0} \r\nPosition: {1}\r\nTag: {2}\r\n---------------",
```

Comment:
Exactly, it's not a black box that is why everything is documented [here ](https://developer.microsoft.com/en-us/windows/holographic/camera_in_unity)which also points to the HoloToolkit Main Camera prefab. We should get developers in the flow of the docs and the toolkit. I am sure you can help with that :)